### PR TITLE
Login component

### PR DESCRIPTION
**Pull request by julienmachon**
Here we go, basically the same as search app:
screenshot from 2018-11-13 11-02-14

### CODE REVIEW

File: src/shared/components/Header/Header.stories.tsx [hunk 1]
```diff
@@ -16,10 +17,12 @@ storiesOf('Components/Header', module)
...
-    name="Mark Hamil"
-    links={[
-      <a href="#">Home</a>,
-      <a href="#">Settings</a>,
-    ]} /></div>,
-));
+  `)(() => (
+    <div style={{ margin: '50px 40px 0px' }}>
+      <Header
+        name="Mark Hamil"
```
Comment (mdupont) [inline src/shared/components/Header/Header.stories.tsx]: Hamill

File: src/shared/components/Login/Login.stories.tsx [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+
+import Login from './index';
+
+storiesOf('Components/Login', module).add(
+  'Login',
+  withInfo(`
+    The Login component allows to redirect to the Auth Provider page.
+
+    ~~~js
+      <Login loginURL="https://keycloak.com?redirect=https://nexus.com" />
```
Comment (mdupont) [inline src/shared/components/Login/Login.stories.tsx]: nexus.com domain already used by something else

File: src/shared/components/Login/Login.test.tsx [hunk 3]
```diff
@@ -0,0 +1,23 @@
+import React = require('react');
+import { shallow, mount } from 'enzyme';
+
+import Login from './index';
+
+const shallowLogin = shallow(<Login loginURL="https://keycloack.com" />);
```
Comment (mdupont) [inline src/shared/components/Login/Login.test.tsx]: keycloak.org (.org, no c)

File: src/shared/components/Header/Header.stories.tsx (hunk 1 above)
Comment (julienmachon) [reply src/shared/components/Header/Header.stories.tsx]: :fearful:

File: src/shared/components/Login/Login.stories.tsx (hunk 2 above)
Comment (julienmachon) [reply src/shared/components/Login/Login.stories.tsx]: It's just an example but I'll use a something we own

**Pull request merged**